### DISCUSSION TITLE
Where does unit 0 fit?

### DISCUSSION THREAD

**Discussion by jdonwells** [TEALSK12/introduction-to-computer-science#309] [Q&A] [answered]
Unit 0 doesn't seem right. Unit 1 is where they set up a Snap! account.

**Comment by DBashirTEALS** [comment, marked as answer]
Hi Don!

Thanks for asking this question.  Unit 0 in the SNAP curriculum does a number of things:
- Allows space for the teacher to set and establish routines and procedures that will hopefully last throughout the school year
- Defines what an algorithm is and the thought process to constructing these to solve simple tasks
- Lightly discussing why programming languages are different
- Constructing the "I Am" outline for the SNAP Self Portrait Project
- Completing the SNAP Self Portrait
- Allowing students to explore SNAP on their own before teaching them CS concepts with the use of SNAP

Students can make their SNAP accts during Unit 0, Lesson 0.4 if you so choose

I hope this response answers your inquiry
